xml/System/ReadOnlySpan`1.xml

```diff
@@ -780,7 +780,7 @@ A `ReadOnlySpan<T>` instance is often used to reference the elements of an array
         <param name="start">The index at which to begin this slice.</param>
         <param name="length">The desired length for the slice.</param>
         <summary>Forms a slice out of the current read-only span starting at a specified index for a specified length.</summary>
-        <returns>A read-only span that consists of <paramref name="length" /> elements from the current span starting at <paramref name="index" />.</returns>
+        <returns>A read-only span that consists of <paramref name="length" /> elements from the current span starting at <paramref name="start" />.</returns>
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentOutOfRangeException">
           <paramref name="start" /> is less than zero or greater than <see cref="T:System.Span`1.Length" />.</exception>
```

Comment:
While you are making changes, this exception message needs to match the on from Span.

It is incomplete.

```text
<paramref name="start" /> or <paramref name="start" /> + <paramref name="length" /> is less than zero or greater than <see cref="T:System.Span`1.Length" />.
```